cirq-core/cirq/contrib/paulistring/convert_to_pauli_string_phasors.py

```diff
@@ -12,18 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import List, Optional, cast, TYPE_CHECKING
+from typing import Optional, cast
 
-import numpy as np
-
-from cirq import ops, protocols, linalg, transformers
+from cirq.contrib.paulistring.clifford_target_gateset import _matrix_to_pauli_string_phasors
```

Comment:
As above - please consider keeping the original private methods instead of importing private function.

Reply:
ditto; not relevant anymore.